integration_test/cloudbuild-integration-test.yaml

```diff
@@ -42,7 +42,9 @@
   id: adapter-main
   entrypoint: 'bash'
   args: [ '-c',
-          "/import/bin/import \
+          "java -Dorg.dcm4che3.imageio.codec.ImageReaderFactory=com/google/cloud/healthcare/imaging/dicomadapter/transcoder/ImageReaderFactory.properties \
```

Comment:
Users having to specify this configuration if they run the adapter from the command line isn't ideal. Why don't the JVM args specified in the javaApplication block in the build file apply here?

Reply:
I expect users to run it via "args:" which invokes entrypoint. In that case there is no need for any extra jvm arguments. (see previous answer for details)

Reply:
I see, so we expect users will only need to run via args. That seems ok.